tools/topology/topology2/cavs-sdw.conf

```diff
@@ -73,6 +73,8 @@ Define {
 	NUM_SDW_AMP_LINKS 0
 	SDW_DMIC 0
 	SDW_JACK true
+	SDW_JACK_FORMAT	''
+	SDW_AMP_FORMAT	''
```

Comment:
@bardliao I have an alternate suggestion instead of empty formats. Why not make the format token a bit mask ie. SDW_AMP_FORMAT "BIT(0) | BIT (1) | BIT(2)" to say that all 3 formats are supported. 

Then in the kernel you can test if all formats are support or not and do a fixup based on what is supported.

Reply:
> @bardliao I have an alternate suggestion instead of empty formats. Why not make the format token a bit mask ie. SDW_AMP_FORMAT "BIT(0) | BIT (1) | BIT(2)" to say that all 3 formats are supported.
> 
> Then in the kernel you can test if all formats are support or not and do a fixup based on what is supported.

2 opens if we use mask.
1. What should kernel select if only s24_le and s16_le is supported, but user play with s32_le? 
2. The format token is a string in IPC3/topology1, if we use a mask in topology2, then topology2 will never be used in IPC3.